sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala

```diff
@@ -620,6 +620,8 @@ class VersionsSuite extends SparkFunSuite with Logging {
     test(s"$version: sql read hive materialized view") {
       // HIVE-14249 Since Hive 2.3.0, materialized view is supported.
       if (version == "2.3" || version == "3.0" || version == "3.1") {
+        // Since Hive 3.0(HIVE-19383), client.runSqlHive is not supported on JDK 11.
+        assume(!(SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_9) && version != "2.3"))
```

Comment:
To claim the following, we should disable all `client.runSqlHive` in our test suites.
> Since Hive 3.0(HIVE-19383), client.runSqlHive is not supported on JDK 11.

However, it seems that we have the followings.
```
$ git grep 'client.runSqlHive'
sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogSuite.scala:    externalCatalog.client.runSqlHive(
sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogSuite.scala:    val bucketString = externalCatalog.client.runSqlHive("DESC FORMATTED db1.t")
sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveShowCreateTableSuite.scala:      .client.runSqlHive(ddl)
sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala:      client.runSqlHive("SET spark.sql.test.key=1")
sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala:        client.runSqlHive("CREATE TABLE indexed_table (key INT)")
sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala:        client.runSqlHive("CREATE INDEX index_1 ON TABLE indexed_table(key) " +
sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala:        client.runSqlHive("CREATE TABLE materialized_view_tbl (c1 INT)")
sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala:        client.runSqlHive(
sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala:        client.runSqlHive(
sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala:        client.runSqlHive(s"DROP INDEX IF EXISTS $indexName ON $tabName")
sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala:      client.runSqlHive(
sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala:            client.runSqlHive(
sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala:            client.runSqlHive(
sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala:            client.runSqlHive(s"ALTER TABLE $db.t ADD COLUMNS (dummy string)")
```

Could you be more specific?

Reply:
How about?
```
// Since Hive 3.0(HIVE-19383), we can not run local mapreduce by `client.runSqlHive` with JDK 11.
```

Reply:
That sounds better~

Reply:
Since it's a comment issue, I'll update them during merging, @wangyum .

Reply:
Oops. Could you change the code like the following?
```scala
- assume(!(SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_9) && version != "2.3"))
+ assume(version == "2.3" || !SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_9))
```

Reply:
Done

Reply:
Thanks!